ssv/committee_runner.go

```diff
@@ -249,29 +250,44 @@ func (cr CommitteeRunner) ProcessPostConsensus(signedMsg *types.PartialSignature
 			sszObject := beaconObjects[validator][root]
 
 			// Submit
-			if role == types.BNRoleAttester {
-				att := sszObject.(*phase0.Attestation)
-				// Insert signature
-				att.Signature = specSig
-
-				if err := cr.beacon.SubmitAttestation(att); err != nil {
-					return errors.Wrap(err, "could not submit to Beacon chain reconstructed attestation")
-				}
-			} else if role == types.BNRoleSyncCommittee {
+			if role == types.BNRoleSyncCommittee {
```

Comment:
why did you switch order?

Reply:
I just thought that it would be better, for the reading flow, to see the duties that are promptly submitted and after it the duties that are stored for later submission.